### ISSUE TITLE
Installation on Raspberry OS/64bit - need guidance

### ISSUE THREAD

**Issue by helarsen** [basler/gst-plugin-pylon#23]
I want to install on Raspberry OS/64bit (Linux pi103 5.15.76-v8+ #1597 SMP PREEMPT Fri Nov 4 12:16:41 GMT 2022 aarch64 GNU/Linux).
As described in the README.md I have installed: https://www.baslerweb.com/en/downloads/software-downloads/software-pylon-7-2-1-linux-arm-64bit-debian/
Also  dependencies (meson and gstreamer) are properly installed for a Debian-based systems as described in the README.md
Finally the files are in /opt/pylon and the export PYLON_ROOT=/opt/pylon is all set

Now here I am lost:
1) The README.md description is a bit unclear from that point onwards and I am stuck in finding the folder in which to invoke the command
_meson builddir --prefix /usr/_ 
and where is the file: _meson_options.txt_. 
Somehow I suppose I am missing a download - but which?

2) I realize that I have downloaded pylon 7.2.1 whereas 7.1 and 6.2 is supported - that could be a problem. Is it? If so how do I uninstall 7.2.1

Thanks for any hints.

**Comment by michaelgruner** [comment]
Hi @helarsen. 

1. You run `meson builder --prefix /usr/` at the root of the gst-plugin-pylon. For example:
```bash
git clone https://github.com/basler/gst-plugin-pylon.git
cd gst-plugin-pylon
meson builddir --prefix /usr/
```
In that same directory you'll find the `meson_options.txt` file, but for Pylon 7 you shouldn't need any option in there.

2. I believe Pylon 7.2.1 should run fine.

**Comment by thiesmoeller** [comment]
documentation fix in v0.5.0
